static/styles/compose.css

```diff
@@ -721,14 +721,9 @@ a.compose_control_button.hide {
         flex-flow: column;
     }
 
-    #compose-textarea {
-        max-height: none !important;
-        flex: 1;
-    }
-
+    #compose-textarea,
     #preview_message_area {
-        max-height: none;
-        height: 1.5em;
+        max-height: none !important;
         flex: 1;
     }
```

Comment:
@timabbott setting ` max-height: none !important;` was aimed to fix that. It avoids it from taking the value from the `max-height` we set via JS code.

<img width="217" alt="image" src="https://user-images.githubusercontent.com/25124304/152303597-149b32a5-32c9-460a-a403-44abc217572a.png">

Reply:
I am not sure how you might have encountered that error state.

Reply:
I can't reproduce it right now, so we may need to wait until someone else encounters it to figure out how to get there.  (Or I suppose it's possible I messed up my testing and found this in the middle of a rebase?  I don't think that's plausible though).